cli/compose/loader/loader.go

```diff
@@ -444,6 +444,8 @@ func LoadVolumes(source map[string]interface{}) (map[string]types.VolumeConfig,
 			if volume.External.Name == "" {
 				volume.External.Name = name
 				volumes[name] = volume
+			} else {
+				return nil, externalVolumeError(name, "name")
```

Comment:
Sorry, I think this should be just a warning, not an error.

You can print a warning with `logrus.Warnf()`